src/lib/transition_frontier_persistence/intf.ml

```diff
@@ -76,10 +83,13 @@ module type Worker = sig
   val deserialize :
        t
     -> root_snarked_ledger:root_snarked_ledger
-    -> consensus_local_state:Consensus.Local_state.t
+    -> consensus_local_state:consensus_local_state
```

Comment:
why this change? consensus isn't a functor, we shouldn't be functoring over bits of it afaik

Reply:
oh, I had some complications hooking in the persistence module into coda_main. Namely, in Coda_main, all the modules were interacting with protocol/coda_pow's `Consesus_mechanism`. As a result, I could not use the already instantiated `Consensus` module.